src/riot-rs-threads/src/lib.rs

```diff
@@ -177,19 +175,8 @@ impl Threads {
 /// # Safety
 ///
 /// This may only be called once.
-///
-/// # Panics
-///
-/// Panics if no thread exists.
 pub unsafe fn start_threading() {
```

Comment:
Hm. `Cpu::start_threading()` is internal. `start_threading()` probably shouldn't be called freely from anywhere. That might be safe now, with the current iteration, for Cortex-M. It might not be in the future, or on other platforms.

Reply:
Is it OK if I just state that despite being pub, this is not supposed to be called by the user, but internally by riot-rs-rt?

Reply:
I tried to convey that this is not a function to meddle with, that it's crafted for special use by `riot-rs-rt` and that it expects some things and probably others.